### PR TITLE
Fix #4848 - Missing Agda.Builtin.Reflection.External from data-files in Agda.cabal

### LINKED ISSUE #4848: Missing Agda.Builtin.Reflection.External from data-files in Agda.cabal
The builtin module `Agda.Builtin.Reflection.External` isn't listed in the data-files in `Agda.cabal`, which means it's not available. This wasn't caught by the test since the test defines `execTC` by itself.

### PR DESCRIPTION
**Pull request by wenkokke**
I think this should fix issue #4848. 

Unfortunately, I can't verify that it works, since I can't run the tests locally.

### CODE REVIEW

**Review by UlfNorell — APPROVED** [review]
Fortunately we have continuous integration to do the tests for you :)

File: test/Succeed/ExecAgda.agda [hunk 1]
```diff
@@ -9,15 +9,11 @@ open import Agda.Builtin.Unit
...
-postulate
-  execTC : String → List String → String → TC (Σ Nat (λ _ → Σ String (λ _ → String)))
-
-{-# BUILTIN AGDATCMEXEC execTC #-}
+open import Agda.Builtin.Reflection.External
 
 macro
   test : Term → TC ⊤
-  test hole = execTC "agda" ("-v0" ∷ "-i" ∷ "test/Succeed" ∷ "test/Succeed/exec-tc/empty.agda" ∷ []) ""
+  test hole = execTC "agda" ("-v0" ∷ "-i" ∷ "test/Succeed" ∷ "exec-tc/empty.agda" ∷ []) ""
```
Comment (gallais) [inline test/Succeed/ExecAgda.agda]: The test was correct. The `Makefile` is called from `../..` hence the need for the relative path.

**Comment by wenkokke** [comment]
Thanks, @gallais, reverted those changes now.

**Comment by wenkokke** [comment]
This should be ready for merging.

**Comment by gallais** [comment]
I have restarted the travis build on the off chance it does not run out of memory this time.
But I agree that if it keeps failing for an unrelated reason this is ready to be merged.

**Pull request merged**